src/modules/personenkontext/domain/personenkontext.ts

```diff
@@ -67,6 +68,7 @@ export class Personenkontext<WasPersisted extends boolean> {
         sichtfreigabe: SichtfreigabeType | undefined = undefined,
         loeschungZeitpunkt: Date | undefined = undefined,
         revision: Persisted<string, WasPersisted> = '1',
+        befristung?: Persisted<Date, WasPersisted>,
```

Comment:
An optional `Persisted<>` is superfluous. `befristung` is in no way dependend on the persisted state of the aggregate, just use `befristung: Date | undefined` (Without the `?` as well)